recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake>=3.12.4
+    - make
+    - boost-cpp>=1.66
```

Comment:
Are we using Boost headers only or are we using Boost libraries as well?

Reply:
I believe just the headers.

Reply:
Do we know what things from Boost are used?

Reply:
I do not. I know it's used by one of the dependencies we build (gunrock) as part of building libcugraph, but I don't know what features it's using. I can look into it though if necessary.

Reply:
Yeah the reason we want to know is we will need to add it to `run` if it does use libraries from `boost-cpp`. This is because `boost-cpp` is one of the exceptions where `run_exports` is not used ( https://github.com/conda-forge/boost-cpp-feedstock/pull/82 ) so needs to be explicitly listed. That said, conda-build will check for missing linkages so should flag these for us. So let's wait to see what CI says